src/main/java/org/folio/orders/events/handlers/ReceiptStatusConsistency.java

```diff
@@ -38,6 +38,7 @@
 public class ReceiptStatusConsistency extends AbstractHelper implements Handler<Message<JsonObject>> {
 
   private static final int LIMIT = Integer.MAX_VALUE;
+  private static final String LANG_QUERY_PARAM = "lang";
```

Comment:
`HelperUtils.LANG` can be used instead of local constant

Reply:
Done